### PR TITLE
add task jmeter

### PR DESCRIPTION
**Pull request by itkg-alavieille**

### CODE REVIEW

File: deployment/tasks/jmeter.cap [hunk 1]
```diff
@@ -0,0 +1,11 @@
+namespace :orchestra do
+    desc 'Jmeter task runner'
+    task :jmeter, :plan_jmx, :args_plan do |task, args|
```
Comment (nicolasThal) [inline deployment/tasks/jmeter.cap]: Can you add some base value for the different argument in the description ?

**Comment by itkg-alavieille** [comment]
@nicolasThal  I added an example but the second (args_plan) argument depends on the test it's the this is the concatenation of scenario options. 

for instance the command for run basic_scenario_page can be 
```
cap inte "orchestra:jmeter[basic_scenario_page.jmx,-Jserver_port='8000' -Jtime_charge='5' ]"
```

**Comment by nicolasThal** [comment]
It is ok if you giving the base argument, after each every one can add all the argument he needs

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**